src/App/App.js

```diff
@@ -148,18 +148,12 @@ class App extends Component {
     });
     newCard.cardCount = 1;
     const newUserCardsData = this.state.userCardsData.concat([newCard]);
-    this.setCardCount(newUserCardsData);
-  }
-
-  removeUserCard = (indexToRemove) => {
-    const [...newUserCardsData] = this.state.userCardsData;
-    newUserCardsData.splice(indexToRemove, 1)
-    this.setCardCount(newUserCardsData);
+    this.saveArray('userCardsData', newUserCardsData);
   }
 
-  setCardCount = (newUserCardsData) => {
-    this.setState({ userCardsData: newUserCardsData },
-      localStorage.setItem('userCardsData', JSON.stringify(newUserCardsData)));
+  saveArray = (arrayName, arrayToSave) => {
+    this.setState({ [arrayName]: arrayToSave },
+      localStorage.setItem(arrayName, JSON.stringify(arrayToSave)));
```

Comment:
These same two lines were being repeated in multiple places, so I made it into a method.

Reply:
Nice!